tests/testthat/test-multitables.R

```diff
@@ -190,6 +190,7 @@ with_mock_HTTP({
 
```

Comment:
I'd recommend that we set up a test with mocks so that we can test this R code quickly without a backend.